src/decision/include/LidarDecision.h

```diff
@@ -48,13 +48,27 @@ class LidarObstacle {
     LidarObstacle(std::vector<Reading> readings);
 
     /**
+    * Gets the distance of the first data from a Obstacle group
+    *
+    * @return the distance of the first data from a Obstacle group
+    */
+    distance_t getFirstDistance();
+
+    /**
+    * Gets the distance of the last data from a Obstacle group
+    *
+    * @return the distance of the last data from a Obstacle group
+    */
+    distance_t getLastDistance();
```

Comment:
Again, the distances are sorted, so perhaps `getMaxDistance` might be a better function name.